stackslib/src/chainstate/stacks/boot/pox-4.clar

```diff
@@ -1332,7 +1332,7 @@
 ;; Set the aggregate public key to the provided value
 ;; TODO: https://github.com/stacks-network/stacks-core/issues/4101
```

Comment:
Don't forget to remove the TODO if its done!

Reply:
Fixed